packages/block-library/src/post-title/index.php

```diff
@@ -11,12 +11,12 @@
  * @return string Returns the filtered post title for the current post wrapped inside "h1" tags.
  */
 function render_block_core_post_title() {
-	global $_experimental_block;
-	if ( ! isset( $_experimental_block->context['postId'] ) ) {
+	$post = gutenberg_get_post_from_context();
```

Comment:
Did we deprecate this function already or not?

Reply:
> Did we deprecate this function already or not?

Not yet. The issue is at #21662, and there's an open pull request at #21696 which would deprecate it, but it's not yet been merged.